core/views.py

```diff
@@ -110,3 +111,38 @@ def post(self: "PostSaveView", request: HttpRequest, pk: int, action_type: str)
         if next_url:
             return redirect(next_url)
         return redirect("post-detail", pk=post.id)
+
+
+class CommunityListView(ListView):
+    model = Community
+    template_name = "core/community-list.html"
+    context_object_name = "communities"
+    paginate_by = 10
+
+
+class CommunityCreateView(LoginRequiredMixin, CreateView):
+    model = Community
+    form_class = CommunityForm
+    template_name = "core/community-create.html"
+
+    def form_valid(self: "CommunityCreateView", form: forms.ModelForm) -> HttpResponseRedirect:
+        form.instance.author = self.request.user
+        response = super().form_valid(form)
+        CommunityMember.objects.create(
+            community=self.object,
+            user=self.request.user,
+            role=CommunityMember.ADMIN,
+        )
+        return response
+
+    def get_success_url(self: "CommunityCreateView") -> str:
+        return reverse_lazy("community-detail", kwargs={"slug": self.object.slug})
```

Comment:
another really minor thing, but we can always improve

https://stackoverflow.com/questions/48669514/difference-between-reverse-and-reverse-lazy-in-django

Reply:
What is your suggestion?

Reply:
Just thought there are reasons why we have two versions of these functions and wanted to use them as intended

Reply:
But I get it. One variant less confusion